types/proposal.go

```diff
@@ -34,7 +34,7 @@ func NewProposal(height int64, round int, blockPartsHeader PartSetHeader, polRou
 	return &Proposal{
 		Height:           height,
 		Round:            round,
-		Timestamp:        time.Now().Round(0).UTC(),
+		Timestamp:        Now(),
```

Comment:
I guess we don't really care about this time for anything? just convenience/debugging ?

Reply:
Proposal message is sort of implementation detail to get ProposalBlock, so we actually don't care about it's timestamp in the context of BFT time. But probably make sense to keep it for debugging.